test/shim.spec.js

```diff
@@ -0,0 +1,30 @@
+import renderer from 'react-test-renderer'
+import { getStorybook } from '../src/storybook-react-client-shim'
+
+describe('Storybook Shim', () => {
+  describe('against Basic Fixture', () => {
+    let ButtonStory, story1, story2
+
+    beforeAll(() => {
+      require('./fixtures/basic')
+      ;[ButtonStory] = getStorybook()
```

Comment:
booo semicolons; make these separate statements (nit)

Reply:
Boo this: 
```
const storyBook = getStorybook()
ButtonStory = storyBook[0]
story1 = ButtonStory.stories[0]
story2 = ButtonStory.stories[1]
```